exporters/trace/zipkin/README.md

```diff
@@ -18,7 +18,7 @@ applications instrumented with Census. The easiest way to start a zipkin
 server is to paste the below:
 
 ```bash
-wget -O zipkin.jar 'https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec'
+curl -sSL https://zipkin.io/quickstart.sh | bash -s
```

Comment:
```suggestion
curl -ssl https://zipkin.io/quickstart.sh | bash -s
```

Reply:
The `-s` and `-S` flags are different. It is not one flag with three letters (perhaps the ordering of them is unfortunate). From `curl --help`:

```
 -S, --show-error    Show error. With -s, make curl show errors when they occur
 -s, --silent        Silent mode (don't output anything)
```

The point being to only output errors from the `curl` command, I believe. Note that this command was copied from the [Zipkin quickstart page](https://zipkin.io/pages/quickstart.html).

Reply:
Sounds good, thanks for the explanation.